### PR TITLE
website/docs: install: add aws

### PR DESCRIPTION
**Pull request by rissson**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by rissson** [comment]
@BeryJu we need to create the secrets for the deploy from https://us-east-1.console.aws.amazon.com/iam/home?region=eu-west-1#/users/details/github-cloudformation-deploy?section=security_credentials

File: website/docs/install-config/install/aws/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: AWS installation
+---
+
+You can install authentik to run on AWS with a Cloudformation template.
```
Comment (fheisler) [inline website/docs/install-config/install/aws/index.md:5]: *CloudFormation (in all instances below)

File: website/docs/install-config/install/aws/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Prerequisites
+
+- An AWS account.
+- An [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) certificate. Take note of the ARN of the certificate.
+
+### Installation
+
+Create a Cloudformation stack [with our template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=authentik&templateURL=https://authentik-cloudformation-templates.s3.amazonaws.com/authentik.ecs.latest.yaml).
```
Comment (fheisler) [inline website/docs/install-config/install/aws/index.md:14]: maybe a note that you already need to be logged into AWS to access this template?

**Review by BeryJu — APPROVED** [review]

File: website/docs/install-config/install/aws/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- An AWS account.
+- An [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/) certificate. Take note of the ARN of the certificate.
+
+### Installation
+
+Log in to your AWS account and create a CloudFormation stack [with our template](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=authentik&templateURL=https://authentik-cloudformation-templates.s3.amazonaws.com/authentik.ecs.latest.yaml).
+
+Under the **Certificate ARN** input, enter the previously created certificate ARN. You can also configure other settings if needed. You can follow the prompts to create the stack.
+
+This stack will create the following resources:
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:18]: For a list, we don't need the period at the end of each one.

File: website/docs/install-config/install/aws/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- AWS SSM secrets for the PostgreSQL user and the authentik secret key.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:20]:
```suggestion
- AWS SSM secrets for the PostgreSQL user and the authentik secret key
```

File: website/docs/install-config/install/aws/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- A VPC for all other resources.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:21]:
```suggestion
- A VPC for all other resources
```

File: website/docs/install-config/install/aws/index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- A RDS PostgreSQL Multi-AZ cluster.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:22]:
```suggestion
- A RDS PostgreSQL Multi-AZ cluster
```

File: website/docs/install-config/install/aws/index.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- An ElastiCache Redis Multi-AZ cluster.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:23]:
```suggestion
- An ElastiCache Redis Multi-AZ cluster
```

File: website/docs/install-config/install/aws/index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+- An ECS cluster with two tasks:
+    - One for the authentik server.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:25]:
```suggestion
    - One for the authentik server
```

File: website/docs/install-config/install/aws/index.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    - One for the authentik worker.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:26]:
```suggestion
    - One for the authentik worker
```

File: website/docs/install-config/install/aws/index.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+- An ALB pointing to the authentik server ECS task with the configured certificate.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:27]:
```suggestion
- An ALB pointing to the authentik server ECS task with the configured certificate
```

File: website/docs/install-config/install/aws/index.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+- An EFS filesystem mounted on both ECS tasks for media file storage.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:28]:
```suggestion
- An EFS filesystem mounted on both ECS tasks for media file storage
```

File: website/docs/install-config/install/aws/index.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+The stack will output the endpoint of the ALB that you can point your DNS records to.
```
Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:30]:
```suggestion
The stack will output the endpoint of the ALB (Application Load Balancer) to which you can point your DNS records.
```

Comment (tanberry) [inline website/docs/install-config/install/aws/index.md:30]:
```suggestion
The stack will output the endpoint of the ALB that to which you can point your DNS records.
```

**Review by tanberry — APPROVED** [review]
Thanks for doc'ing this!

**Pull request merged**